bitnami/elasticsearch/templates/metrics-deploy.yaml

```diff
@@ -48,9 +52,11 @@ spec:
             {{- if .Values.metrics.extraArgs }}
             {{- toYaml .Values.metrics.extraArgs | nindent 12 }}
             {{- end }}
+          {{- end }}
           ports:
             - name: metrics
               containerPort: 9114
+          {{- if not .Values.diagnosticMode.enabled }}
```

Comment:
Maybe the user want to have metrics enabled in diagnosticMode.
What do you think?

Reply:
The problem is that we can't guarantee the metric container is going to work if the service in the main container is not up and running. The idea of this mode is to provide a way for users to access the different containers. Being said that, it can be improved in the future based on the feedback provided by the users or internally, this is just a first iteration to see how it works and if it's useful

Reply:
OK